src/oauth-handler.js

```diff
@@ -31,7 +32,9 @@ export default function oauth(Client, {
   shipCache = null
 }) {
   function getURL(req, url, qs = { token: req.hull.token }) {
-    return `https://${req.hostname}${req.baseUrl}${url}?${querystring.stringify(qs)}`;
+    const host = `https://${req.hostname}${req.baseUrl}${url}`;
+    if (qs === false) return host;
```

Comment:
I think this is what I would expect from function like this - if I pass empty qs it will not append only the "?" char at the end.